app/api/v2/handlers/ability_api.py

```diff
@@ -22,35 +22,47 @@ def add_routes(self, app: web.Application):
         router.add_patch('/abilities/{ability_id}', self.update_ability)
         router.add_delete('/abilities/{ability_id}', self.delete_ability)
 
-    @aiohttp_apispec.docs(tags=['abilities'])
+    @aiohttp_apispec.docs(tags=['abilities'], summary='Get all abilities.',
+                          descriptions='Provides a list of all available abilities.')
     @aiohttp_apispec.querystring_schema(BaseGetAllQuerySchema)
     @aiohttp_apispec.response_schema(AbilitySchema(many=True, partial=True))
     async def get_abilities(self, request: web.Request):
         abilities = await self.get_all_objects(request)
         return web.json_response(abilities)
 
-    @aiohttp_apispec.docs(tags=['abilities'])
+    @aiohttp_apispec.docs(tags=['abilities'], summary='Get an ability.',
+                          description='Provides one ability based on its ability id.')
```

Comment:
For the endpoints that have a unique id in the url, can you add the parameter description? It can be done by adding the following to the `docs` decorator:
```
parameters=[{
      'in': 'path',
      'name': 'ability_id',
      'schema': {'type': 'string'},
      'required': 'true',
      'description': 'UUID of the Ability to be retrieved'
  }])
```

Reply:
Done.